src/yhdistys.jade

```diff
@@ -0,0 +1,120 @@
+    link(rel='icon', type='image/png', href='icons/favicon-160x160.png', sizes='160x160')
+    link(rel='icon', type='image/png', href='icons/favicon-96x96.png', sizes='96x96')
+    link(rel='icon', type='image/png', href='icons/favicon-16x16.png', sizes='16x16')
+    link(rel='icon', type='image/png', href='icons/favicon-32x32.png', sizes='32x32')
+    link(rel='shortcut icon', href='icons/favicon.ico')
+    link(rel='icon', href='icons/favicon.ico')
+    meta(name='msapplication-TileColor', content='#10558c')
+    meta(name='msapplication-TileImage', content='icons/mstile-144x144.png')
+    meta(property='og:image', content='images/logo.png')
+    script(src='https://js.stripe.com/v3/')
+    script(src='//use.typekit.net/scb5xny.js')
+    script.
+      try{Typekit.load();}catch(e){};
+
+  body
+    .site
+      .container
+        .header
+          .header__container
+            .header__nav
+              a(href='/') etusivu
+              a(href='/yhdistys.html') yhdistys
+            .header__headline
+              .header__logo
+              h1.header__title Koodiklinikka ry
+          .header__members
+            #members
+
+        .content
+          section
+            .row
+              .bread
+                .column.column1-2
+                  h3 Rekisteröity yhdistys
+                  p.
+                    Koodiklinikka on nyt rekisteröity yhdistys!
+                  p.
+                    Lähes kolmivuotisen taipaleensa aikana Koodiklinikka on kasvanut lähes tuhannen rekisteröityneen jäsenen yhteisöiksi ja näin saavuttanut aseman Suomen suurimpana ohjelmointiaiheisena yhteisönä! Liittymällä Koodiklinikka ry:n jäseneksi tuet toimintaamme, ja tulevaisuudessa yhdityksen jäsenyys oikeuttaa etuihin Koodiklinikan tapahtumissa.
+                  p.
+                    Yhdistyksen jäsenyys <b>ei ole</b>, eikä tule olemaan, <b>pakollinen</b> Koodiklinikan käyttäjille tai meetuppeihin osallistujille.
+                .column.column1-2
+                  .bread-img
+            .row
+              .bread
+                .column.column1-2
+                  h3 Liity jäseneksi
+                  #membership-form.form
+
+                .column.column1-2
+                  p.
+                    Kuka tahansa Slack:iin rekisteröitynyt käyttäjä voi liittyä yhdistyksen jäseneksi maksamalla kertaluontoisen <b>10€</b> jäsenmaksun.
+                  p.
+                    Nimellisellä jäsenmaksulla tulemme tekemään tapahtumistamme vieläkin mielenkiintoisempia ja kattamaan kustannuksia jäsenmäärämme kasvaessa.
+                  p.
+                    Jos tarvitset Slack-kutsun, siirry <a target="_blank" href="/">etusivulle</a>.
+                  p.
+                    Jäsenrekisterin rekisteriseloste luettavissa <a target="_blank" href="https://docs.google.com/document/d/1w-NZkf1EFYMC8a4LsoZouhpMVKORmPi8UFWRhurr0oM/edit?usp=sharing">Google Drivessä</a>.
+
+
+      footer
```

Comment:
Could we have these bits in another jade file that we could import to both views?